@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -1356,8 +1354,15 @@ export class TileGeometryCreator {
      * Creates and add a background plane for the tile.
      * @param tile - Tile
      * @param renderOrder - Render order of the tile
+     * @param checkIfValid - Whether to add a check to see if this is necessary to add.
```

Comment:
Why would one add an invalid background plane?

Reply:
Good point, I didn't want to have the check for the BackgroundDataSource, but it is anyway correctly configured because it sets `addGroundPlane`, so it will pass this check. Will remove